source/documentation/runbooks/aws-credentials.html.md.erb

```diff
@@ -0,0 +1,33 @@
+---
+title: AWS credentials remediation process
+last_reviewed_on: 2021-11-16
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+These are the steps to be followed if we learn a set of AWS credentials have been compromised. 
+
+Make sure the relevant user/service are made aware and establish a channel of communication with them. Jointly with the user, evaluate the best course of action to retire the credentials.
+
+## The credentials can be retired immediately
+
+1. Locate the user in the IAM section of the AWS console
+2. Delete all keys and also disable the password if there is one.
+3. Attach a `AWSDenyAll` policy to the user while remediation is in progress
+	- Click on the user from the IAM Users Screen
+	- In the Permissions tab, click Add Permission
+	- Select "Add policy" from the 3 tabs on the page
+	- Find the AWSDenyAll policy and apply it to the user
+4. Investigate CloudTrail logs for activity from that user in case any resources have been created or other relevant activity after the time you were made aware of the compromise.
+5. Once you feel safe to do so, force the password reset on the user and have them issue themselves new keys.
+
+
+## Retiring the credentials immediately will lead to service downtime
+
+1. Agree an approach with the affected service team. **This needs to be done quickly.** If key decision makers are not available at the time, the security of the estate and the service take precendence over downtime.
+2. Based on the agreed approach, you might re-issue a new key first before deleting the old one. 
+3. In all cases, keep a close eye on the CloudTrail logs for any activity coming from the compromised credentials so you can immediately detect and potentially shut down unwanted activity. 
+
+
+**In all cases this is a security incident and so the security team should be made aware of it. The service team/user might need to additional remediation based on advice from Security.**
```

Comment:
Do we need to add that the Service Team takes responsibility for notifying Security of the breach?

Reply:
We should also include link to the form for reporting incidents for info.

Reply:
Done aaaand done!